radix-engine-utils/Cargo.toml

```diff
@@ -4,12 +4,19 @@ version = "0.9.0"
 edition = "2021"
```

Comment:
Minor: I'd suggest we rename `radix-engine-untils` to `radix-engine-profiling`, since a few people have been confused by this. :)